business/products.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Providers of machine translation services
+---
+
+# Providers
+
+Machine translation **providers** offer ready-to-use machine translation systems. Typically the systems are provided as APIs, and also as applications or integrations.
+
+They vary by:
+
+- language support
+- accuracy
+- price
+- customization options
+- data confidentiality
+
+and other features, as well as underlying architecture.
+
+
+The advantage of using a machine translation provider is that the service is already working and can be integrated immediately. The alternative is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
+
+Unlike raw models, production machine translation systems usually provide language identification and sentence segmentation.
+
+Providers offer machine translation services via a platform, an API or both. Some providers offer limited services for free or paid full services.
+
+| Company | Product | Service |
+| ----------- | ----------- | ----------- |
+| Google | Google Translate | Character-based self-serve API |
+| LingvaNex | Translation API | Character-based self-serve API |
+| Yandex | Translate API | Character-based self-serve API |
+| Microsoft | Translator Text API | Character-based self-serve API |
+| Amazon | Translate API | Character-based self-serve API |
+| ModernMT | API | Character-based self-serve API |
+| IBM | Watson Language Translator | Character-based self-serve API |
+| Baidu | 翻译API | Character-based self-serve API |
+| DeepL | API | Character-based self-serve API |
+| Alibaba | Machine Translation | Character-based self-serve API |
+| Systran | Systran Translate Pro | Character-based self-serve API |
+| Unbabel | Unbabel | Volume- and content-based [hybrid translation](hybrid-translation.md) suscription |
```

Comment:
Seems it's called "Unbabel Translate"

https://www.google.com/search?q=%22unbabel+translate%22&client=firefox-b-d&sxsrf=AOaemvIx9VGVpAEgNMBqNUt40RLvhdi_yg%3A1631789609791&ei=KSJDYZfGL-n-7_UP_qWrwAI&oq=%22unbabel+translate%22&gs_lcp=Cgdnd3Mtd2l6EAMyBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIICAAQCBAHEB46BwgAEEcQsANKBAhBGABQv0tY71FgslJoAnACeACAAVmIAd8EkgEBOJgBAKABAcgBA8ABAQ&sclient=gws-wiz&ved=0ahUKEwjX_qeAqoPzAhVp_7sIHf7SCigQ4dUDCA0&uact=5